src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_buffer_fusing.cpp

```diff
@@ -419,6 +423,18 @@ void prepare_buffer_fusing::run(program& p) {
                     //  crop output buffer
                     //  |_low_pad_|__data_size__|___|<-upper pad
 
+                    //  feature num of pad should be accumulated if dep has been optimized out.
+                    auto& dep = node.get_dependency(0);
+                    if (dep.is_type<crop>() && dep.can_be_optimized()) {
+                        auto dep_pad = dep.get_output_layout().data_padding;
+                        if (dep_pad.lower_size().batch[0] == 0 && dep_pad.upper_size().batch[0] == 0 &&
+                            dep_pad.lower_size().spatial[0] == 0 && dep_pad.upper_size().spatial[0] == 0 &&
+                            dep_pad.lower_size().spatial[1] == 0 && dep_pad.upper_size().spatial[1] == 0) {
+                            opt_lower_pad += dep_pad.lower_size().feature[0];
+                            opt_upper_pad += dep_pad.upper_size().feature[0];
+                        }
```

Comment:
what if it does not go into this `if` condition? Should we throw exception?
Could you also add a unit test for this case?